src/app/container/container.component.html

```diff
@@ -30,14 +30,14 @@
       (tool?.mode === ModeEnum.AUTODETECTQUAYTAGSAUTOMATEDBUILDS || tool?.mode === ModeEnum.AUTODETECTQUAYTAGSWITHMIXED) && !tool?.gitUrl
     "
   >
-    <mat-card class="alert alert-info" role="alert">
+    <mat-card class="alert alert-info mat-elevation-z" role="alert">
```

Comment:
If this `mat-elevation-z` is for all `alert-info` mat-cards, then it should be added to the `alert-info` class so that future uses of `alert-info` will automatically have this style. Can investigate using the `mat-elevation` [mixin](https://v6.material.angular.io/guide/elevation#mixins) or just using `box-shadow: none`

Reply:
Looked into `mat-elevation` mixin, few caveats:
- can only do elevation from 0 - 24 (not z) so its not possible to have no shadow with this method
- it applies the `box-shadow` style to the component, but without the class `mat-elevation-z` it gets overridden by the general `mat-card` shadow style (see below)
- doesn't seem to allow `!important`  but either way style doesn't work and the `!importants` would clash

I thought about `box-shadow: none !important;` when implementing but we have a problem with conflicting overrides in shadow styles by the general `mat-card` shadow style again (see below), so it doesn't work either

**General mat-card shadow style:**
https://github.com/dockstore/dockstore-ui2/blob/0007b986539ab3883bf72b9239d165e64038a725/src/styles.scss#L1315-L1318

So the only method/ probably most robust method that I found is to just include the `mat-elevation-z` class at every instance and avoid the overrides completely (`alert-warning` and `alert-danger` also follow the same method from when I updated them last)

Reply:
> can only do elevation from 0 - 24 (not z) so its not possible to have no shadow with this method

I think `mat-elevation-z` is the `0` value (no elevation)

Thanks for looking into it. It doesn't seem possible without a bit of refactoring. I think it would still be good to document that all banners should have `mat-elevation-z` so that it has a flat appearance. Perhaps you can add something to https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide about the banners?

Reply:
> I think mat-elevation-z is the 0 value (no elevation)

Yeah I thought that too, but `mat-elevation(0)` adds a shadow style so I don't think they correspond in this case? Which is interesting...
| ![image](https://user-images.githubusercontent.com/97123241/223254598-45b493e8-aa9a-40c0-918b-cb190a04c5af.png) | ![image](https://user-images.githubusercontent.com/97123241/223254900-31e8e3b3-15f0-4e17-8deb-840f1d81230e.png) |
|:--:|:--:|

>  Perhaps you can add something to https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide about the banners?

Makes sense 👍️ I'll add a note in the style guide

Reply:
Note added!